7.analyze-coverage/coverage-analysis.py

```diff
@@ -3,36 +3,29 @@
 Interpret Compression
 7.analyze-coverage/coverage-analysis.py
 
-Track and visualize gene set activity across compressed features
+Track gene set coverage across compressed features
```

Comment:
What do you mean by coverage per "GO pathway" or "Cancer module"?  I am thinking of how much information is reconstructed but not sure how this concept relates to pathway information

Reply:
Ah, good point. I updated the documentation in the next commit!

Basically, we want to see the proportion of genesets that are captured (or "covered") by the specific algorithm implementation.